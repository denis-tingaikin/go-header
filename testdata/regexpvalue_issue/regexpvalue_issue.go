// A 2020
// B
// A 2021
// B
// TEXT

package regexpvalue
