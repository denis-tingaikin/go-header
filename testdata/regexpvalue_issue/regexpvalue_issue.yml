template: |-
  {{.COPYRIGHT_HOLDER}}TEXT

values:
  const:
    'YEAR': '2020'
  regexp:
    '.COPYRIGHT_HOLDER': |-
      (A {{ .YEAR }}
      (.*)
      )+
