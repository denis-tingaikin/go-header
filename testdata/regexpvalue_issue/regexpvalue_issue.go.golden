// A 2020
// B
// A 2021
// C
// TEXT

package regexpvalue
